gridt/src/app/movements/movements.page.spec.ts

```diff
@@ -50,4 +52,14 @@ describe('MovementsPage', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+<<<<<<< HEAD
+  it('should dismiss the alert when leaving the page', () => {
+=======
+
+  it("should dismiss the alert when leaving the page", () => {
+>>>>>>> fe1feac8813acb9cbd3248540d796f9397e807e6
```

Comment:
You did not really deal with this conflict I am getting the impression

Reply:
Aperantly not, no...